product_end_of_life/views/product_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<odoo>
+    <!-- Product Template form view -->
+    <record id="view_template_property_form_eol_view" model="ir.ui.view">
+        <field name="name">view.template.property.form.eol.view</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.view_template_property_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='group_lots_and_weight']" position="inside">
+                <field name="eol_date"/>
+            </xpath>
+        </field>
+    </record>
+    <!-- Product Variant form view -->
+    <record id="product_variant_easy_edit_eol_view" model="ir.ui.view">
+        <field name="name">product.variant.easy.edit.eol.view</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_variant_easy_edit_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='weight']" position="after">
+                <group></group>
```

Comment:
@tbaden The empty group is needed?

Reply:
@nikul-serpentcs as I did not change this part I'm not 100% sure, but most likely for a correct placement inside the view.

Reply:
@tbaden Ok, Can you please check if need then keep as it is, otherwise remove it.

Thanks